### DISCUSSION TITLE
DataTable: Editable cell with p:selectonemenu not moving focus properly with tab key

### DISCUSSION THREAD

**Discussion by crcatedr** [primefaces/community#3146] [PrimeFaces] [answered]
Hello,

I have an editable datatable similar to the 'Cell Editing with Click and RightClick' example in the Primefaces showcase. When pressing tab to move through the columns, focus works fine until a cell with a p:selectonemenu.  On tab out, focus doesn't move to the next column. If you use a h:selectonemenu component it works fine, which can also be seen in the showcase example.

I found an old issue that describes what I'm experiencing perfectly: https://github.com/primefaces/primefaces/issues/2378. It appears to have been fixed back then, but it doesn't seem to be working for me? Any help would be appreciated.

For reference, using Primefaces 14.0.7.

Thanks!

**Comment by melloware** [comment]
ahh my guess is something changed with `input.parent().hasClass('ui-helper-hidden-accessible')` is not true anymore with the new structure of SelectOneMenu.  Can you open a GitHub issue for this bug?

I will fix it for 14.0.9

**Reply by crcatedr** [reply, marked as answer]
Issue created: https://github.com/primefaces/primefaces/issues/12989

Thanks!
